gems/pending/util/miq_apache/miq_apache.rb

```diff
@@ -206,7 +206,7 @@ def self.create_balancer_config(opts = {})
     def self.create_redirects_config(opts = {})
       opts[:redirects].to_miq_a.each_with_object("") do |redirect, content|
         if redirect == "/"
-          content << "RewriteRule ^/self_service(?!/(assets|images|img|styles|js|fonts)) /self_service/index.html [L]\n"
+          content << "RewriteRule ^/self_service(?!/(assets|images|img|styles|js|fonts|bower_components|gettext)) /self_service/index.html [L]\n"
```

Comment:
What it really means is redirect everything that's not our files to index.html so that angular's router can pick it up... So maybe a simple `RewriteCond` testing for the file existence would be enough... Except doing it like that may expose files that shouldn't really be..